src/instruments/src/EFB/Settings/Settings.tsx

```diff
@@ -103,6 +65,127 @@ const AircraftConfigurationPage = () => {
         }
     };
 
+    return (
+        <div className="bg-navy-lighter rounded-xl px-6 shadow-lg divide-y divide-gray-700 flex flex-col">
+
+            <div className="py-4 flex flex-row justify-between items-center">
+                <span className="text-lg text-gray-300">Thrust Reduction Altitude (ft)</span>
```

Comment:
These should be heights, not altitudes. The FMC adds these heights to the departure runway altitude to get the altitude... e.g. default thrust reduction alt = (departure runway altitude) + (default thrust reduction height setting)